migrations/migrations/20210308144235-user-model.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
```

Comment:
bdd table name must be singular. For convention.

Reply:
ok